hypergpufunctions.cu

```diff
@@ -201,36 +269,32 @@ void HyperFunctionsGPU::oneD_array_to_mat(int* img_array, int cols, int rows, in
     {
         for (int j=0; j<rows; j++)
         {
-
-                Vec3b color = classified_img.at<Vec3b>(Point(0,0));
-                color[0]=img_array[val_it] ;
-                color[1]= img_array[val_it+1] ;
-                color[2]=img_array[val_it+2] ;
-                //cout<<color<<endl;
-                classified_img.at<Vec3b>(Point(k,j))= color   ;
-                val_it+=3;
-
+            Vec3b color = classified_img.at<Vec3b>(Point(0,0));
```

Comment:
can we do the same casting here as we did for greyscale images?

Reply:
added